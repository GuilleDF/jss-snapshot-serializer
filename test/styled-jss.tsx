import React from 'react';
// @ts-ignore
import styled from 'styled-jss';
import { render } from '@testing-library/react';
import serializer from '../src/jss-snapshot-serializer';

expect.addSnapshotSerializer(serializer);

const RedBox = styled('div')({
  width: 100,
  height: 100,
  backgroundColor: 'red',
});

describe('styled-jss', () => {
  it('removes non-deterministic class names', () => {
    const redBox = render(<RedBox />).asFragment();
    expect(redBox).toMatchSnapshot();
  });
});
