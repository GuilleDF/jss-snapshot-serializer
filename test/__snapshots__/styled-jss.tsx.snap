// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`styled-jss removes non-deterministic class names 1`] = `
<DocumentFragment>
  <div
    class="div-1-0-1-1"
  />
</DocumentFragment>
`;
